src/vi/hentaicube/src/eu/kanade/tachiyomi/extension/vi/hentaicube/HentaiCB.kt

```diff
@@ -58,6 +59,24 @@ class HentaiCB :
 
     override val altNameSelector = ".post-content_item:contains(Tên khác) .summary-content"
 
+    override fun popularMangaFromElement(element: Element): SManga {
+        val manga = SManga.create()
```

Comment:
call `super.popularMangaFromElement(element).apply {`` and make changes to thumbnail